Tests/Shared.Specs/StringAssertionSpecs.cs

```diff
@@ -1773,6 +1774,222 @@ public void When_the_actual_string_equivalent_to_the_expected_but_with_trailing_
 
         #endregion
 
+        #region ContainAll
+
+        [Fact]
+        public void When_null_collection_containall_should_throw_argument_exception()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Act
+            //-----------------------------------------------------------------------------------------------------------
+            Action act = () => "a".Should().ContainAll(null);
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Assert
+            //-----------------------------------------------------------------------------------------------------------
+            act
+                .ShouldThrow<ArgumentException>()
+                .WithMessage("Cannot assert string containment of values in null collection");
```

Comment:
I prefer to use wildcards in `WithMessage` so that only the relevant parts are verified. Also see point 10 from http://www.continuousimprover.com/2015/11/12-tips-to-write-unit-tests-that-dont.html